defaults/main.yml

```diff
@@ -12,6 +12,9 @@ beacon_node_build_script_path: '{{ beacon_node_path }}/build.sh'
 beacon_node_user: 'nimbus'
 beacon_node_group: 'staff'
 
+beacon_node_validators_dir: ""
+beacon_node_secrets_dir: ""
```

Comment:
If those are undefined please comment them out with "value" as `~`, or whatever is the default used by the node.

Reply:
I prefer to not specify the default here, because the logic for deriving the default value is present in the beacon node binary and it could change.

Reply:
In other words, when the empty string is specified the parameter is not supplied on the command-line at all (enabling Nimbus to use its default value).

Reply:
I didn't mean specify a default. I mean COMMENT IT OUT and THEN specify a default.

Reply:
Something like this:
```yaml
# Currently these are relative to --data-dir, but that could change.
#beacon_node_validators_dir: '{{ beacon_node_data_path }}/{{ beacon_node_data_folder }}/validators'
#beacon_node_secrets_dir: '{{ beacon_node_data_path }}/{{ beacon_node_data_folder }}/secrets'
```